superset/assets/src/visualizations/wordcloud/buildQuery.ts

```diff
@@ -0,0 +1,14 @@
+import buildQueryContext, { FormData, QueryContext } from 'src/query';
+
+export default function buildQuery(formData: FormData): QueryContext {
```

Comment:
do you think it's worthwhile to specify specialized `FormData` interfaces per-vis so it's clear which (sub-set of) fields a given vis requires/depends on?

Reply:
Good suggestion! 

Fiddling with this to see if we can compose the interface definitions (much like the query builders).